beacon-chain/rpc/eth/validator/structs.go

```diff
@@ -3,9 +3,13 @@ package validator
 import "github.com/prysmaticlabs/prysm/v4/beacon-chain/rpc/eth/shared"
 
 type AggregateAttestationResponse struct {
-	Data shared.Attestation `json:"data"`
+	Data shared.Attestation `json:"data" validate:"required"`
 }
 
 type SubmitContributionAndProofsRequest struct {
-	Data []shared.SignedContributionAndProof `json:"data"`
+	Data []shared.SignedContributionAndProof `json:"data" validate:"required"`
+}
+
+type SubmitAggregateAndProofsRequest struct {
+	Data []*shared.SignedAggregateAttestationAndProof `json:"data" validate:"required"`
```

Comment:
this is a pointer while the other above are not, i think we should just keep the pointers